subiquity/common/filesystem/manipulator.py

```diff
@@ -220,6 +220,7 @@ def partition_disk_handler(self, disk, spec, *, partition=None, gap=None):
                     spec['size'], gap.size)
                 spec['size'] = gap.size
 
+        gap = gap.split(spec['size'])[0]
```

Comment:
I wonder how many callers of split are going to care about the remainder gap? I guess we'll find out...

Reply:
:smile: 
The remainder actually gets used with the core commits for use_gap.  It works extremely well for creating multiple partitions in one gap, which is why I started `gap.split`.

Reply:
Ah right, that makes sense!